smac/runhistory/runhistory.py

```diff
@@ -21,7 +21,13 @@
 class RunKey(collections.namedtuple('RunKey', ['config_id', 'instance_id', 'seed', 'budget'])):
     __slots__ = ()
 
-    def __new__(cls, config_id, instance_id, seed, budget=0.0):
+    def __new__(
+        cls,
```

Comment:
If there is no type for `cls` it would at least be great to have a docstring explaiing what this is

Reply:
I'll add that.